target/linux/arc770/image/Makefile

```diff
@@ -24,6 +24,9 @@ endef
 
 define Device/nsim
 	$(call Device/vmlinux)
+	DEVICE_VENDOR := Synopsys
+	DEVICE_MODEL := SDP board
+	DEVICE_VARIANT := AXS101
```

Comment:
So, axs101 or nsim?

Reply:
nSIM refers to a simulation engine that can also run this image.
I could add it as an alternative model. What do you think?

Ref.: https://github.com/openwrt/openwrt/commit/576621f1e353339fe81ece6cff7e8bcd12cbc7b8

Reply:
E.g.:
```
DEVICE_ALT0_VENDOR := Synopsys
DEVICE_ALT0_MODEL := nSIM
```

Reply:
well, if the image is named like it, why not just go with

DEVICE_VENDOR := Synopsys
DEVICE_MODEL := nSIM

(or nSIM 700)?

Reply:
AXS101 sounds like the appropriate model name.
If we have to use nSIM, then how about:

```
DEVICE_VENDOR := Synopsys
DEVICE_MODEL := nSIM
DEVICE_ALT0_VENDOR := Synopsys
DEVICE_ALT0_VARIANT := AXS101
```

Reply:
My problem is that I simply have no idea how to verify that this is actually specifically "AXS101" ...

Reply:
I got the identifier "AXS101" from the commit that introduced support for this model.

My main objective is to set at least one set of DEVICE_* variables. The need for completeness comes second for me.

Reply:
This commit https://github.com/openwrt/openwrt/commit/30d75720faa8d9e92454705e15d5f9ebcb9d0c6a

suggests that 
```
DEVICE_VENDOR := Synopsys
DEVICE_MODEL := nSIM
```

would be the correct approach. Let's go with that then.